### ISSUE TITLE
Invalid CPU type

### ISSUE THREAD

**Issue by pavm035** [uber/mockolo#195]
I have two issues to report

1. When i try to use binary directly on Intel machine it's throwing Bad CPU type, After inspecting found that this binary is only arom64 compatible and missing x86 support, Would you please consider adding support that x86(intel) architecture?

2. Try to build docker image from this source code but it failed due to one of the system framework os.signpost not available in docker image in linux machine

**Comment by mithatsk** [comment]
I also had the Bad CPU type error but building the binary on the local machine and using that build fixed this error for me. Following the below instructions on the home page should help with it.

```shell
git clone https://github.com/uber/mockolo.git
cd mockolo
swift build -c release
.build/release/mockolo -h  // see commandline input options below
```

After running above commands you can copy the executable and move it to your project folder.

**Comment by uhooi** [comment]
@pavm035
Sorry, this issue should have been fixed by #200 here, and I would like you to try again.

**Comment by sidepelican** [comment]
1. Maybe solved at #200.
2. At `2.0.0`, build on Linux are supported. Please try on it.

Please feel free to reopen this if you still have trouble.
